src/Test/FixCommonErrors/FixCommonErrorsTest.cs

```diff
@@ -3632,9 +3632,19 @@ public void RemoveSpaceBetweenNumbersSpaceTest()
         {
             var engine = new RemoveSpaceBetweenNumbers();
             var sub = GetGenericSub();
-            sub.Paragraphs.First().Text = "<i>their 1 000-year sleep</i>";
+            sub.Paragraphs.First().Text = "<i>their 1 232-year sleep</i>";
```

Comment:
Update the hundreds to be diff then 000 since it can now be any digit from 0-9 for all 3 digits